csunplugged/topics/models.py

```diff
@@ -11,7 +11,7 @@ class GlossaryTerm(models.Model):
 
     #  Auto-incrementing 'id' field is automatically set by Django
     slug = models.SlugField(unique=True)
-    term = models.CharField(max_length=200, unique=True)
+    term = models.CharField(max_length=200)
```

Comment:
why shouldn't a term be unique? What if there is an `algorithm.md` with heading `Algorithm` but I don't notice and create a `algorithms.md` with heading `Algorithm`. Now I have identical terms with different slugs which seems silly.